folktables/acs.py

```diff
@@ -65,6 +65,9 @@ def get_definitions(self, download=False):
         return load_definitions(root_dir=self._root_dir, year=self._survey_year, horizon=self._horizon,
                                 download=download)
 
+def fillna_safe(x, value=-1):
```

Comment:
Can you describe what the intended behavior of this function should be? 

Note that `np.nan_to_num` will convert to nans to `0.0` unless you set the keyword `nan=value`. See [here](https://numpy.org/doc/stable/reference/generated/numpy.nan_to_num.html#numpy-nan-to-num).

So, currently, I believe this function will first convert all nans to zero and then pass an array without any nans to pandas.

Am I getting this wrong?